src/test/java/com/google/api/codegen/testdata/php_main_library.baseline

```diff
@@ -401,9 +401,17 @@ class LibraryServiceClient
      *                              that don't use retries. For calls that use retries,
      *                              set the timeout in RetryOptions.
      *                              Default: 30000 (30 seconds)
-     *     @type string $appName The codename of the calling service. Default 'gax'.
-     *     @type string $appVersion The version of the calling service.
-     *                              Default: the current version of GAX.
+     *     @type string $libName The API library software used for calling the service.
+     *                            (Unless you are writing an API client itself, leave this as default.)
+     *     @type string $libVersion The API library software version used for calling the service.
+     *                               (Unless you are writing an API client itself,
+     *                               leave this as default.)
```

Comment:
We could actually leave `$libName/$libVersion` undocumented, or discourage their use in some other way. Unless we recognize the `$libName` on the server side, we won't do anything with these values.